ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -2023,6 +2023,7 @@
       {
         "UserRoles": [
           "[APPLICANTSOLICITORONE]",
+          "[RESPONDENTSOLICITORTWO]",
```

Comment:
Why should solicitor2 have access to respondent1LitigationFriend?

Reply:
Respondent2 needs be able to see the respondent1 lit details in the claim details screen. So I gave them read access.